packages/webview_flutter_lwe/CHANGELOG.md

```diff
@@ -121,7 +125,7 @@
 
 * Update Dart and Flutter SDK constraints.
 * Update Flutter and Samsung copyright information.
-* Update webview_flutter_tizen to use platform view interface.
+* Update webview_flutter_lwe to use platform view interface.
```

Comment:
It's a log of the past. Do you need to fix it?

Reply:
I think it's okay because it's history. 
but on the other hand, I am wondering if I should reset the version and history of one of the two webview plugins.
For example, reset the version of webview_flutter_lwe to 0.1.0 and start a new history of changes. Do you think it's okay?

Reply:
i think the version should be determined according to the deprecate status of [webview_flutter_tizen] (https://pub.dev/packages/webview_flutter_tizen).
When this pr will merged, isn't webview_flutter_lwe package re-registered(?) in pub.dev? If so, I think, the version should be 0.1.0.
But if not, I think this change log should be left because it is a history of '0.2.0'.

Reply:
General rule: The changelog of a package should match the versions listed in the "Versions" tab on pub.dev.

This means that this package (webview_flutter_lwe) should have an empty changelog and the version should be either 0.1.0 or 1.0.0 since it's being released for the first time.